### ISSUE TITLE
duplicate entries of "worker_rlimit_nofile" in nginx.conf

### ISSUE THREAD

**Issue by sonontse** [rtCamp/easyengine#499]
This is on a fresh install of debian 7.0 and current version of easyengine.  After creating the stack with "ee stack install --web" and running the command "ee site create example.com" it gives the following error.

2015-04-20 12:44:23,957 (DEBUG) ee : nginx: [emerg] "worker_rlimit_nofile" directive is duplicate in /etc/nginx/nginx.conf:4
nginx: configuration file /etc/nginx/nginx.conf test failed
2015-04-20 12:44:23,958 (INFO) ee : [Failed]

When inspecting nginx.conf, multiple entries of worker_rlimit_nofile is found causing nginx to fail to start.

**Comment by josemejiasji** [comment]
same here :/

**Comment by sonontse** [comment]
Also happens when updating a existing site to HHVM.  It will make a copy of the mysql, move ee-config.php and htdocs to the backup folder and fail due to multiple lines of "worker_rlimit_nofile".  Would have to revert all the moved files and delete the duplicate line in nginx.conf in order to fix it.

**Comment by gau1991** [comment]
Hello,
This issue is fixed in EasyEngine 3.1.1

**Comment by bamajr** [comment]
I got a very, very similar error, with EasyEngine 3.1.1 today, on a brand new installation of Debian GNU/Linux 7 (with backports)...

```
nginx: [emerg] "worker_rlimit_nofile" directive is duplicate in /etc/nginx/nginx.conf:4
nginx: configuration file /etc/nginx/nginx.conf test failed
```

I started with a fresh Debian 7 Install. I set the root password. I ran apt-get update/upgrade.

I installed ee with:

```
ee stack install --web
```

I then installed the utilities with:

```
ee stack install --utils
```

I then installed the administrative utilities with:

```
ee stack install --admin
```

...Yes, before you say "That is redundant" let me identify that I know it is redundant. I'm simply spelling out how I got this error.

The error presented itself first, after the admin install. Then presented itself again, when I ran:

```
nginx -t && service nginx reload
```

I've attached a screenshot of my I/O from SSH shell so you can see it for yourselves. Hopefully it helps.

![screen shot 2015-04-24 at 12 18 43 pm](https://cloud.githubusercontent.com/assets/1583597/7324738/9d17176c-ea7e-11e4-9b29-a558fb25f0d0.png)
